WarehouseKing.java

```diff
@@ -0,0 +1,22 @@
+import Java.util.ArrayList; 
+
+public class WarehouseKing{
+	
+	private ArrayList <Store> StoreBranches;
+
+	public String storeFinder(String productName){
+		String desiredStores = "These are the stores that have" + productName + "available: "; 
+		for(Store s: StoreBranches){
+			if(s.hasProduct(productName)){
+				desiredStores += s.getName(); 
+				desiredStores += ", "; 
+			}
+		}
+		return desiredStores; 
+	}
+
+	public static void main(String[] args) {
```

Comment:
This program isn't running. It just initializes the stores. Please allow for some functionality for user. For example,
if I input the number 1 I want to view the inventory of store #1